src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -3,13 +3,19 @@
 import by.onliner.core.driver.DriverManager;
 import by.onliner.core.elements.widget.Button;
 import by.onliner.core.elements.widget.Text;
+import by.onliner.core.elements.widget.TextInput;
 import by.onliner.core.utils.ActionsHelper;
 import by.onliner.core.utils.Waiter;
 import io.qameta.allure.Step;
 import org.openqa.selenium.support.FindBy;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class ShoppingCartPage extends AbstractPage {
 
+    private static final String priceRegexPattern = "^[0-9]+";
```

Comment:
Please make sure that the constant is named properly. Constants should be upper cases

Reply:
Fixed